README.md

```diff
@@ -26,6 +40,28 @@ This is the dependency you need:
 Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto)
 for more technical details.
 
+Create a `hello.txt` file with `"Hello World!"` content on file-system-based
```

Comment:
@Sammers21 maybe it's better to prepend this paragraph with a `Examples of usage` header?